src/lib.rs

```diff
@@ -35,11 +35,13 @@
 #![cfg_attr(test, deny(warnings))]
 #![cfg_attr(test, deny(missing_docs))]
 
-#[macro_use] extern crate serde_derive;
-#[macro_use] extern crate serde_json as json;
 extern crate chrono;
 extern crate reqwest;
 extern crate serde;
+#[macro_use]
```

Comment:
Please remove this change, I prefer `#[macro_use]` on the same line as when it's sorted with vim's `:sort` function it will place all macro_use statements above the rest, whcich help prevents macros missing errors because they were defined after they were required.